### DISCUSSION TITLE
Regarding IGV issue consultation

### DISCUSSION THREAD

**Discussion by wuguosssssfd** [community/community#119938] [Discussions]
### Select Topic Area

Question

### Body

![e004a2471285361d4c25b0c6b9bc8ef3](https://github.com/community/community/assets/167315055/763644b6-51a8-4246-9743-0038611d03d9)
When using the latest version of IGV, I encountered an issue where the software view cannot be zoomed out to the minimum. When I try to minimize the view, I face difficulties. Our input files are in BAM format, and the window size can only be minimized to 27kb and cannot be further reduced to above 40kb, as shown in the screenshot below. Adjusting the window tracks does not resolve this issue, which is inconvenient for my data analysis and visualization.

I would like to inquire whether this issue is due to a limitation of the software or if it is an error in my configuration or usage. If it is a software issue, have you already identified this problem and addressed it in future updates? Are there better solutions available? Thank you for your assistance.

**Comment by ettaboyle** [comment]
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, **including posting the same comment in multiple places.** As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
